Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/MainActivity.kt

```diff
@@ -37,13 +37,15 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.main_activity)
 
-        authController = AuthController()
+        authController = AuthController
```

Comment:
Does it make sense to maintain a reference to an object? Or should we just call the methods statically? I don't really know the answer to this.

Reply:
nice catch..definitely not...was in too much of a rush yesterday :-(
No need to maintain a ref to a singleton...this broke unit tests on UserProfileTest too :-(